app/services/background.py

```
@@ -66,40 +66,94 @@ def __init__(self):
+        for attempt in range(max_retries):
+            try:
+                self.services[service_name] = "running"
+                self.logger.info(f"✅ {service_name} service started successfully")
+                
+                # Run the service function
+                await service_func()
+                
+            except asyncio.CancelledError:
+                # Re-raise cancellation immediately - don't treat as error
+                raise
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)